src/containers/Secrets/Secrets.js

```diff
@@ -59,6 +59,18 @@ const initialState = {
   toBeDeleted: []
 };
 
+function base64Decode(input) {
+  return decodeURIComponent(
+    atob(input)
+      .split('')
+      .map(
+        character =>
+          '%' + ('00' + character.charCodeAt(0).toString(16)).slice(-2) // eslint-disable-line prefer-template
```

Comment:
I think a comment here explaining this goodness might be useful for maintaining later - looks like awesome magic! Kudos for getting things working, definitely a great improvement

Reply:
yeah string encoding is always fun 😂 

added some comments